docs/gaia/gaia.rst

```diff
@@ -92,15 +92,51 @@ degrees around an specific point in RA/Dec coordinates.
   >>> r = Gaia.query_object_async(coordinate=coord, width=width, height=height)
   >>> r.pprint()
 
-           dist             solution_id     ...       ecl_lat
-                                            ...      Angle[deg]
-  --------------------- ------------------- ... -------------------
-  0.0026029414438061079 1635378410781933568 ... -36.779151653783892
-  0.0038537557334594502 1635378410781933568 ... -36.773899692008634
-  0.0045451702670639632 1635378410781933568 ... -36.772645786277522
-  0.0056131312891700424 1635378410781933568 ... -36.781488832325074
-                  ...                 ... ...                 ...
-  Length = 152 rows
+           dist             solution_id     ... epoch_photometry_url
+                                            ...
+  --------------------- ------------------- ... --------------------
+  0.0026034636994048854 1635721458409799680 ...
+  0.0038518741347606357 1635721458409799680 ...
+    0.00454542650096783 1635721458409799680 ...
+   0.005613919443965546 1635721458409799680 ...
+   0.005846434715822121 1635721458409799680 ...
+   0.006209042666371929 1635721458409799680 ...
+   0.007469463683838576 1635721458409799680 ...
+   0.008202004514524316 1635721458409799680 ...
+   0.008338509690874027 1635721458409799680 ...
+   0.008406677772258921 1635721458409799680 ...
+                    ...                 ... ...                  ...
+    0.01943176697471851 1635721458409799680 ...
+   0.019464719601172412 1635721458409799680 ...
+   0.019467068628703368 1635721458409799680 ...
+   0.019752561500226976 1635721458409799680 ...
+    0.01991656886177004 1635721458409799680 ...
+   0.020149589233310516 1635721458409799680 ...
+   0.020307185970548904 1635721458409799680 ...
+   0.020454730686780127 1635721458409799680 ...
+   0.020802655215768254 1635721458409799680 ...
+   0.021615117161838747 1635721458409799680 ...
+  Length = 50 rows
+
+The query returns a result containing 50 rows by default. The number of rows returned can be controlled by setting ``Gaia.ROW_LIMIT`` appropriately.
```

Comment:
Maybe factor out the default value as if it changes, it's difficult to catch in the narrative docs 

```suggestion
Queries return a limited number of rows controlled by ``Gaia.ROW_LIMIT``. To change the default behaviour set this appropriately. 
```